src/exodus/exoduswallet/test/wallet_tests.cpp

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2019 The Zcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <boost/test/unit_test.hpp>
+
+#include "wallet/wallet.h"
+#include "wallet/test/wallet_test_fixture.h"
+
+#include "../wallet.h"
+
+struct ExodusSigmaTestingSetup : WalletTestingSetup
+{
+    ExodusSigmaTestingSetup()
+        : WalletTestingSetup()
```

Comment:
We don't need this line due to it implicit do for us.

Reply:
nice catch